docs/tools/hedgehog-engine/rangers/files/cemt.md

```diff
@@ -0,0 +1,9 @@
+---
+description: Cemt tools for Sonic Frontiers
+---
+# Cemt Tools
+
+## Effect Color Tool
+Editor tool for working with Sonic Frontiers cemt files.
+
+[Download :fontawesome-solid-download:](https://cdn.discordapp.com/attachments/987489054377508924/1043923384179884052/effect_color_tool.rar){ .md-button .md-button--primary}
```

Comment:
Please do not use Discord to link to the tool. Either use the assets system that's described in the README.md of HedgeDocs, or link directly to the tools' page